packages/teams/test/utils/get-user-creatable-teams.test.ts

```diff
@@ -60,50 +61,62 @@ describe("getUserCreatableTeams", () => {
         "portal:user:addExternalMembersToGroup"
       ]
     };
+
     expect(
       getUserCreatableTeams(user, "premium", "9.1", "In House").length
     ).toBe(5, "Premium, all privs, should be 5 creatable team types");
+
     expect(getUserCreatableTeams(user, "basic", "9.1", "In House").length).toBe(
       3,
       "Basic, all privs, should be 3 creatable team types"
     );
+
     expect(
       getUserCreatableTeams(user, "portal", "9.1", "In House").length
     ).toBe(3, "Portal, all privs, should be 3 creatable team types");
+
     // remove updateGroups...
     user.privileges = [
       "portal:user:createGroup",
       "opendata:user:designateGroup",
       "portal:user:addExternalMembersToGroup"
     ];
+
     expect(
       getUserCreatableTeams(user, "premium", "9.1", "In House").length
     ).toBe(4, "Premium, all privs, should be 5 creatable team types");
```

Comment:
This message seems wrong... user does not have all privs, and only 4 groups can be created